### DISCUSSION TITLE
What are record field modifiers?

### DISCUSSION THREAD

**Discussion by Mr-M1M3** [pocketbase/pocketbase#3331] [Q&A]
## Sorry, for noticing late. What are record field modifiers as marked in this screenshot of the docs:

![screenshot](https://github.com/pocketbase/pocketbase/assets/78679538/03404559-a53d-41c5-8f64-a73ae21e8e00)

**Comment by ganigeorgiev** [comment]
They are not documented very well at the moment (_this will change in the next release with the addition of `fields` functions/modifiers and there will be an extra section under the Web APIs guides_), but these modifiers are part of the field key and could be used to modify an existing field value without being explicit about its old/previous state.

For example, let's assume we have a "total" `number` field and we want to increment it with `2`:

```js
let record = await pb.collection("example").getOne("RECORD_ID")

// old value: 10
console.log(record.total)

record = await pb.collection("example").update(record.id, {
    // the same as total:12
    'total+': 2
})

// new value: 12
console.log(record.total)
```
